### PR TITLE
VCDAT 

### PR DESCRIPTION
**Pull request by SandiSamuel**
In vcdat I sorted the files, graphic methods and templates and I change the buttons for closing files

### CODE REVIEW

File: conda-recipe/vcdat/meta.yaml [hunk 1]
```diff
@@ -20,4 +20,4 @@ source:
   git_rev: v0.0.1
```
Comment (doutriaux1) [inline conda-recipe/vcdat/meta.yaml]: @SandiSamuel we should change this to master  or . some new tag point since we changed the version tag

Comment (mattben) [inline conda-recipe/vcdat/meta.yaml]: instead of `version: 0.0.2` you want `version: master`? I can make that change in the PR.

Comment (doutriaux1) [inline conda-recipe/vcdat/meta.yaml]: either that or we create a `0.0.2` tag point on current master. It's better to have it match.

Comment (mattben) [inline conda-recipe/vcdat/meta.yaml]: Do you want me to tag a new release in vcdat master 0.0.2? before or after we merge this? Does @SandiSamuel need to change anything in the file before I merge? Conda Recipes are new to me.

Comment (doutriaux1) [inline conda-recipe/vcdat/meta.yaml]: I don't really care, you guys own the repo, I just think the conda recipe and the repo versioning should be in sync. 
Basically, anything released to the public should have an easily identifiable tag. If you want I can make this recipe follow the format of all of our other repos and auto push every successful `master` build to `nightly` conda label. Then you guys only need to worry about tagging for official releases.

Comment (mattben) [inline conda-recipe/vcdat/meta.yaml]: @doutriaux1 that would be greatly appreciated. Then we can always have a working master/nightly. Thank you

**Pull request merged**